requirements.txt

```diff
@@ -0,0 +1,5 @@
+argparse
```

Comment:
Required true but a default package of pyton, isn't it ?

Reply:
Corrected in https://github.com/XanX3601/deep_sentiment140/pull/3/commits/4833fdfb78d9b6d5d7314cc5c237879d8ad4c360.